### DISCUSSION TITLE
The False Prophet - New Megafauna?

### DISCUSSION THREAD

**Discussion by TallDeekus** [shiptest-ss13/Shiptest#463] [Ideasguy]
_People really seemed to like this idea, so I am actually putting an idea here for once..._

### **The False Dragon - A short story to start with...**
You are mining alone, as you are used to in these lonesome voyages, being the only one awake you figured getting more fuel from a rock in space would help shake off the sickly feeling cryo leaves in the lower reaches of your gut.

Swing after swing, breath after breath, you discipline the stone into new shapes, until a large crack forms ahead lit dimly by your flashlight, and barely large enough to fit a Kepori.

Something lurks beyond it however, this is not unusual, you've dealt with your fair share of goliaths and ridden a few, you approach the crack in the wall shining your flashlight through it, confidently resting an abused crowbar over your shoulder, little did you know this was your last mistake.

Beyond the threshold of the stone, in a chamber glazed with greed and pulsating matter, you see a large creature, as tall as a Drake, and as long as a Space Dragon, but it is... Wrong, like it didn't belong in this world, it was Draconic in shape, but that is where it's similarities ended, the creature lacked scales, instead its coarse flesh was like tough hide, and had a color much like that of a Mi-go, but more sinister was the snout, flat and almost crooked, the jaw hangs unhinged and dressed with sharp needle-like teeth, the tail is easily twice the length of the body, constantly coiling around the monster like an anxious snake, two appendages that are a mockery of what wings stand for, more like clubs with vicious spines like a porcupine adorned on the end, and its eyes... 

When you gaze upon the eyes it has you, your flashlight and crowbar float from your hands as your gaze is captured by the starry void its eyes should be, you feel a numb chill enter your mind as a command rings out from an otherworldly tongue that causes your nose to bleed...

### _**"Come closer."**_

Your body moves immediately, and you are helpless when you crawl through the crack in the wall, dislocating your shoulder on the way through, alas, the pain was the last thing you remembered.

### Actual Information - Statistics and Trauma
The False Prophet, Fake Dragon, Sighteater, Faux Savior, it goes by many names yet it is unclear where such a thing came from.

Considering the story above explains the description well, I won't repeat it here, I will move on to a more gameplay standpoint.

The False Prophet is supposed to be a Shiptest unique Megafauna that encourages miners to be more careful of where they look, it's basic attributes are as follows:

- Slightly less health then an Ash Drake.
- 100% Burn Damage Taken.
- 80% Brute Damage Taken.
- Doesn't Breathe.
- Has 'Flight' (Can move through space, although it will usually appear in enclosed spaces)
- Can use doors.
- Can Speak Common, Draconic, and [DATA EXPUNGED].
- Is stunned by Bluespace Crystals (And various other things that use Bluespace)
- Melee Attacks deal Brute, and deals a bit less damage then an Ash Drake.

The Eyes of a False Dragon have _Hypnotic_ properties that it uses to entrance prey long enough for it to either capture them with its tail, or consume them, in gameplay this can do the following things...

1. Inspecting the False Prophet multiple times within the span of 4 Minutes causes the inspector to enter a **Stunned** state that leaves them briefly open to suggestion, either from the Fake Dragon's random babbling it does on occasions, or to nearby beings, the **Stun** doesn't knock down, and lasts twice as long on simplemobs.

2. The False Prophet will sometimes stop in place to stare at whatever it is attacking, if the attacker mouses over the False Prophet during this brief but noticeable time, they will be **Stunned** as if they met condition 1 above, however this stun lasts for twice as long.

3. Whenever condition 1 or 2 would apply, the mob receives 15 brain damage and a negative moodlet with the text "IT IS IN MY HEAD!", the Fake Dragon will also immediately target the stunned individual.

4. If the player would receive a brain trauma from this, it is always the one that causes Hypnotic Stupors.

5. Players can only be stunned by condition 1 or 2 once every 7 seconds, however the Moodlet and the Brain Damage still apply.

### Actual Information - Spoils and Spoiled

"We all love Eldritch Horror but what do we get for this?" - You, probably.

Glad you asked, you see, it wouldn't be a Megafauna without it's rewards, here are some of the rewards...

1. Its **Flesh**, which oddly heals brain trauma when cooked, don't ask why, but it can't be used for recipes.

2. The **Eyes**, the very things you were avoiding before, if surgically put into your face-hole, will give you the effects of them! In short, people who look into your eyes can be entranced as if you were a False Prophet!

3. The **Spines** are a **Crusher** reward, which can be attached as a Trophy, this causes the Crusher to do minor damage and stun creatures nearby when you hit a marked creature! This damage is brute for simple mobs and the stun lasts for 1-2 seconds, on Humanoids it deals Stamina Damage and has a slow for a second, it does nothing to robots...

4. Its **Tail**, which can be skinned into a cape, and the resulting hideless tail can be cooked into a Roast!

5. **Precious Minerals**, mostly Diamond and Uranium.

6. A rare chance to vomit up a **Dead Ashwalker with a bag of gold.**

This is unfortunately the end of the idea, however it is open to suggestions and ideas!

_No miners were harmed in the making of this idea._

### Right so, update and balance.

There is currently a working prototype of the creature, the code is mostly done it just needs balance, sprites, and perhaps audio(?), footage and images are provided below, please excuse the typos, neither I nor Mark noticed before taking the picture.

https://user-images.githubusercontent.com/92000398/155083037-db75d6a6-9903-4ef3-a9ed-120bdd404a17.mp4
![False Prophet Moodlet](https://user-images.githubusercontent.com/92000398/155083148-db51bbc3-7904-42d5-90e3-708b80659cc6.png)
![False Prophet Dialogue](https://user-images.githubusercontent.com/92000398/155083452-757a05e1-65f3-4904-90fe-e4f42575a59b.png)

As shown in the video it looks exceptionally dangerous, especially when alone, **while this is nice and all, this isn't the intended difficulty...**

Below are some suggested ideas to balance out the mechanics of the Megafauna:

- A 'Dead By Daylight' styled heartbeat aura, to warn players when they are near it to give them some chance.
- Have the boss have a sort of 'Frenzy' gauge, this was suggested by @MarkSuckerberg, this idea consists of having the boss be in various states, a 'calm' state and a 'frenzied' state, while in the calm state the boss moves slower, uses psychological and mental attacks more, and moves with purpose, while in a frenzied state, they move in an aggressive yet clumsy fashion, resort to physical attacks (like a tail swing AOE) more often, use their gaze attacks less often, and move faster, states change with the flow of combat, and the beast is likely to enter a Frenzy when it realizes it's normal tricks aren't working on it's prey, likewise, when things calm down it starts to retreat to it's calm state, and heals determined by how much Frenzy it loses (not alot of healing though!).
- A 'Tick' system, making it so the first incident of brainwashing/hallucination is minor, but each successive incident is more severe.
- Odd weaknesses suggested by users from the discord, like tinfoil hats and reflective surfaces, perhaps some medical drugs (or even recreational drugs, hey, you can't be mentally fucked with if you are on LSD.) can also work.

Here are some proposed changes to make the boss behave different from normal Megafauna:

- Have it actively hunt players, I am not talking about normal aggro, I am talking about actual predator stalking prey sort of deal, the monster stays away from ruins and ships, however if it detects prey nearby it will slowly 'stalk' the player, never getting too close, but close enough to warrant it a threat, this goes well with the Aura and Frenzy ideas above.
- Have it visible through walls via Meson Glasses but look like a normal Ashdrake, however, if you can see it... It knows where you are, this pairs well with the Stalking idea above, and perhaps can work even through cameras.
- Invisible to Borgs and AI, but only when it is completely 'Calm', if it is Frenzied it becomes visible to them, allowing them to assist the miner in need!

**Comment by TallDeekus** [comment]
Edited the story because there were some minor things that Irked me, like my obsession with commas...

**Comment by ilikesaying** [comment]
That tail meat better be amazing

**Reply by TallDeekus** [reply]
> That tail meat better be amazing

On it's own raw only a human could stomach it (Following the themes that humans are creatures that are willing to eat anything), cooked though, it is a dish worthy of a feast, like a super bite burger but with a tinge of victory.

**Reply by ilikesaying** [reply]
Also dorfs

**Reply by TallDeekus** [reply]
Fair, fair.

**Reply by TallDeekus** [reply]
Honestly Fauna need more stuff like cookables, and while I understand why you can't cook things like Bubblegum, I would totally cook an Ice Drake into Gumbo.

**Reply by ilikesaying** [reply]
Lavaland Roast

**Comment by Martinpachu** [comment]
i agree there should be more planet food

**Comment by TallDeekus** [comment]
Right so, it has been a while so let me update on the status of this idea.

**A semi-working prototype is made, but it is horribly unbalanced, I'll be updating the idea above with suggested balance changes.**
